docs/querying/sql-functions.md

```diff
@@ -866,28 +889,78 @@ Returns a union of HLL sketches.
 
 ## HUMAN_READABLE_BINARY_BYTE_FORMAT
 
-`HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])`
+Converts an integer byte size into human-readable [IEC](https://en.wikipedia.org/wiki/Binary_prefix) format.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `HUMAN_READABLE_BINARY_BYTE_FORMAT(value[, precision])`
+* **Function type:** Scalar, numeric
+
+<details><summary>Example</summary>
+
+  Converts `1000000` into IEC format.
```

Comment:
```suggestion
  The following converts `1000000` into IEC format.
```
Retain "the following for consistency"

Reply:
Changed it to "The following example..."

Reply:
as well as in similar instances